### DISCUSSION TITLE
Web Interface for GPT-Engineer

### DISCUSSION THREAD

**Discussion by SumitKumarDev10** [AntonOsika/gpt-engineer#481] [General]
I want to build a website for **gpt-engineer** with **flask** and **bootstrap**

**Comment by SumitKumarDev10** [comment]
I need help in designing the website, what webpages it will have and functionality it will exhibit. I have a few ideas as follows:-
    Index Page
    About Us Page
    Getting Started Page
    Documentation Page
    Blog Page
    Community Page
    I am taking [Pandas Website](https://pandas.pydata.org/) as an inspiration.

**Reply by SaiPanneerselvam** [reply]
Nice! We will see where we can use it in the project.

**Reply by SaiPanneerselvam** [reply]
I will see with the rest of our group. 🥇 👍 😃

**Comment by SaiPanneerselvam** [comment]
> I need help in designing the website, what webpages it will have and functionality it will exhibit. I have a few ideas as follows:- Index Page About Us Page Getting Started Page Documentation Page Blog Page Community Page I am taking [Pandas Website](https://pandas.pydata.org/) as an inspiration.

@SumitKumarDev10, I will be able to do this in html designs and code, but i still need the area we are building the website confirmed.

**I am now wondering also if this repo is still a project because it looks like no one is talking or making.**

**Comment by SaiPanneerselvam** [comment]
@SumitKumarDev10 you are now added to the Official Discussion Thread for The Web Interface Project and the Official Repository.
There are reasons that it is called official, ask me why.

**Reply by SumitKumarDev10** [reply]
Thank You!

**Reply by SaiPanneerselvam** [reply]
your'e welcome!

**Comment by SaiPanneerselvam** [comment]
Anyone else who would like to be added to the Repository can ask me in discussion #505.

**Reply by PierrunoYT** [reply]
You can use cursor.sh for dev

**Reply by Richjerk** [reply]
Add me please 🙏

**Reply by SaiPanneerselvam** [reply]
please use #505 for anything else. Thanks.

**Comment by mbouande** [comment]
J'ai besoin d'aide pour la création d'un site web E-commerce svp
